tasklist/importer-870/src/main/java/io/camunda/tasklist/zeebeimport/v870/processors/common/UserTaskRecordToTaskEntityMapper.java

```diff
@@ -185,6 +191,58 @@ public Map<String, Object> getUpdateFieldsMap(
             case "dueDate" -> updateFields.put(TaskTemplate.DUE_DATE, entity.getDueDate());
             case "followUpDate" ->
                 updateFields.put(TaskTemplate.FOLLOW_UP_DATE, entity.getFollowUpDate());
+            case "priority" -> updateFields.put(TaskTemplate.PRIORITY, entity.getPriority());
+            default -> {
+              LOGGER.warn(
+                  "Attribute update not mapped while importing ZEEBE_USER_TASKS: {}", attribute);
+            }
+          }
+        }
+      }
+      default -> {}
+        // TODO handle update of other fields in https://github.com/camunda/tasklist/issues/4306
+    }
+    return updateFields;
+  }
+
+  public Map<String, Object> getUpdateFieldsListViewMap(
```

Comment:
💭  I think we can get rid of Tasklist List View updates for 8.7, but this can be also be done in a follow-up issue. 
We are raising the possibility to delete this index, once it was incorporate on `tasklist-task`
(applicable for task and variables case)

Reply:
Good point. I would leave as it is with this PR, and it's up to the HTO team to decide if the index and everything related to it should be deleted in a separate branch.